source/retryable-writes/retryable-writes.rst

```diff
@@ -409,6 +409,9 @@ error should be raised instead as the caller can infer that an attempt was made
 and the second error is likely more relevant (with respect to the current
 topology state).
 
+If a retry attempt fails with an error labeled NoWritesPerformed, drivers MUST
+return the original error.
+
```

Comment:
Can you bump the spec version and add a changelog entry?

Reply:
Sorry for not requesting this before I approved. I assumed @prestonvasquez would get to it when ticking the checkboxes in the issue template.